src/node.js

```diff
@@ -42,26 +42,29 @@ try {
  *
  *   $ DEBUG_COLORS=no DEBUG_DEPTH=10 DEBUG_SHOW_HIDDEN=enabled node script.js
  */
+exports.loadInspectOpts = function () {
```

Comment:
Your redefinition of inspectOpts is a breaking change, and thus won't be accepted. Please try to find another way to do what it is you're trying to do.